src/shared/components/palette/collection/write-collection-field.component.ts

```diff
@@ -103,6 +112,35 @@ export class WriteCollectionFieldComponent extends AbstractFieldWriteComponent i
     return index ? `${this.caseField.label} ${displayIndex}` : this.caseField.label;
   }
 
+  isNotAuthorisedToCreate() {
+    if (this.isExpanded) {
+      return false;
+    }
+    return !this.profile.user.idam.roles.find(role => !!this.caseField.acls.find( acl => acl.role === role && acl.create === true));
```

Comment:
Can we extract `acl.role === role && acl.create === true` to a separate method like `hasCreateAccess`.

Reply:
👍 for the double bang operators 😄